### ISSUE TITLE
Feature request: show more details to a pull request

### ISSUE THREAD

**Issue by cheesi** [hanneswidrig/azure-devops-pull-request-center#26]
It would be nice, if the pull request center would show a few more details about the pull requests, as Azure DevOps is doing it by themself.

A sample screenshot from the pull request view of a repo:
![image](https://user-images.githubusercontent.com/5590254/190015583-4948ba7f-76c1-465f-99a2-4476b32ac2af.png)

The following things would be nice to show in the pull request center:
- any updates on a PR since the last visit (bold title + blue line on the left side)
- any tags on the PR

**Comment by hanneswidrig** [comment]
These features are possible, but would require contribution as I don't have the time to work on it right now. I also haven't been working with ADO in several years making it hard to verify I haven't broken things when I make changes.

**Comment by cheesi** [comment]
I have tried to take a look into it in the past already, it seems to use an undocumented Contribution API. I guess it would not be easy to implement.

**Comment by hanneswidrig** [comment]
Yeah that's been my experience with a lot of the ADO APIs, they've always lagged behind and there was basically no official documentation beyond the TypeScript types on the npm package I use to query the API.
